leetcode/0003.Longest-Substring-Without-Repeating-Characters/3. Longest Substring Without Repeating Characters.go

```diff
@@ -26,25 +26,35 @@ func lengthOfLongestSubstring(s string) int {
 	return result
 }
 
-// 解法二 滑动窗口
-func lengthOfLongestSubstring_(s string) int {
-	if len(s) == 0 {
-		return 0
+// 解法二 滑动窗口-数组桶
+func lengthOfLongestSubstring1(s string) int {
```

Comment:
This solution is the wrong answer. If s = " ", this solution output 0, but actually expected 1.